### DISCUSSION TITLE
2 Karten auf einer Seite ?

### DISCUSSION THREAD

**Discussion by sfx12** [Tastaturberuf/anyStores#92] [Feature Requests / Funktionswünsche]
Ich würde gerne 2 Karten auf einer Seite laden, am besten im Tab/Reiter.

es wird jedoch immer nur die erste Karte geladen ?

da es keine "Filterfunktion" der Marker gibt, hatte ich gedacht das dies der einfachste und schnellste Weg wäre, der jedoch nicht funzt :-(

**Comment by Tastaturberuf** [comment]
Das ist im Moment nicht möglich, da die ID der Map noch nicht eindeutig ist und dann doppelt existiert.

**Comment by sfx12** [comment]
hmm, shit, gib t es evtl ein "hardcoded" hack ?

Mir gehts drum eigentlich um die Filterung und dies wäre der einfachste weg !!

**Comment by Tastaturberuf** [comment]
Das wird nicht gehen, weil auch das Javascript dann mit dieser ID reden muss. Im Moment müssen zwei Seiten geladen werden oder vielleicht mit nem iFrame irgendwie gefrickelt.
